### PR TITLE
[FLINK-31556][docs-zh] Translate "rest" page of "sql-gateway" into Chinese

### PR DESCRIPTION
**Pull request by FangYongs**
## What is the purpose of the change

This PR aims to translate "rest" page of "sql-gateway" into Chinese

## Brief change log

  - Translate "rest.md" page of "sql-gateway" into Chinese

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (yes / no) no
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (yes / no) no
  - The serializers: (yes / no / don't know) no
  - The runtime per-record code paths (performance sensitive): (yes / no / don't know) no
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn, ZooKeeper: (yes / no / don't know) no
  - The S3 file system connector: (yes / no / don't know) no

## Documentation

  - Does this pull request introduce a new feature? (yes / no) no
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

**Comment by FangYongs** [comment]
Tests failed due to kafka connector which is not related to this PR

**Comment by FangYongs** [comment]
@flinkbot run azure

File: docs/content.zh/docs/dev/table/sql-gateway/rest.md [hunk 1]
```diff
@@ -94,15 +91,15 @@ Endpoint Options
 REST API
 ----------------
 
-The available OpenAPI specification is as follows. The default version is v2.
+OpenAPI 规范如下，默认版本是 v2。
 
 | Version                       | Description |
 | ----------- | ------- |
 | [Open API v1 specification]({{< ref_static "generated/rest_v1_sql_gateway.yml" >}}) | Allow users to submit statements to the gateway and execute. |
```
Comment (fsk119) [inline docs/content.zh/docs/dev/table/sql-gateway/rest.md:98]: 这两句也可以翻译一下

File: docs/content.zh/docs/dev/table/sql-gateway/rest.md [hunk 2]
```diff
@@ -123,9 +120,9 @@ The OpenAPI specification is still experimental.
 Data Type Mapping
 ----------------
 
-Currently, REST endpoint supports to serialize the `RowData` with query parameter `rowFormat`. REST endpoint uses JSON format to serialize
-the Table Objects. Please refer [JSON format]({{< ref "docs/connectors/table/formats/json#data-type-mapping" >}}) to the mappings.
+目前 REST endpoint 支持使用查询参数 `rowFormat` 序列化 `RowData`。REST endpoint 使用 JSON 序列化 Table Objects。
+请参考 [JSON format]({{< ref "docs/connectors/table/formats/json#data-type-mapping" >}}) 查看映射关系。
```
Comment (fsk119) [inline docs/content.zh/docs/dev/table/sql-gateway/rest.md:124]: 查看映射关系 -> 查看类型映射关系？

File: docs/content.zh/docs/dev/table/sql-gateway/rest.md [hunk 3]
```diff
@@ -123,9 +120,9 @@ The OpenAPI specification is still experimental.
 Data Type Mapping
 ----------------
 
-Currently, REST endpoint supports to serialize the `RowData` with query parameter `rowFormat`. REST endpoint uses JSON format to serialize
-the Table Objects. Please refer [JSON format]({{< ref "docs/connectors/table/formats/json#data-type-mapping" >}}) to the mappings.
+目前 REST endpoint 支持使用查询参数 `rowFormat` 序列化 `RowData`。REST endpoint 使用 JSON 序列化 Table Objects。
```
Comment (fsk119) [inline docs/content.zh/docs/dev/table/sql-gateway/rest.md:123]: Table Object -> Table 对象？

**Review by fsk119 — COMMENTED** [review]
Thanks for your contribution. It looks good to me in general. Please also translate the description about rest options at the line 68.

**Comment by FangYongs** [comment]
Thanks @fsk119 , updated

**Review by fsk119 — APPROVED** [review]
LGTM

**Pull request merged**